### PR TITLE
feat: getting started page

### LINKED ISSUE #754: Design - Get Started page
Development issue for Design getting started page.
Design issue: #545

Mostly Gatsby components except for the data table and in the inline filter. 

We are still updating the video so that will be delivered in a following sprint. 

[Figma redlines](https://www.figma.com/file/KZ6SWR2abIniIJY4RewDeA/Getting-Started-Design---Redline?node-id=10%3A13436)

[Links for cards and data table](https://ibm.box.com/s/v687a4c86xbr69j59j1llpaevw2gwu96)

Images:

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #754 

## Testing:  
-  http://localhost:3000/designing/get-started

TODOs: 
- [x] Some links have a gray link design
- [x] Some paragraphs seem to have a condensed view?
- [x] Merge in data table #756  --> [PR](https://github.com/carbon-design-system/carbon-platform/pull/842)

### CODE REVIEW

File: services/web-app/pages/assets/designing/get-started/index.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+<Row className="resource-card-group">
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Get Figma"
+      aspectRatio="2:1"
+      actionIcon="launch"
+      href="https://gatsby.carbondesignsystem.com"
+      >
+
+![Figma icon](/designing/get-started/figma.svg)
```
Comment (alisonjoseph) [inline services/web-app/pages/assets/designing/get-started/index.mdx:46]:
```suggestion
![Figma icon](/designing/get-started/figma.svg)
```
<MdxIcon name="figma" />

File: services/web-app/pages/assets/designing/get-started/index.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Get Sketch"
+      actionIcon="launch"
+      aspectRatio="2:1"
+      href="https://gatsby.carbondesignsystem.com"
+      >
+
+![Sketch icon](/designing/get-started/sketch.svg)
```
Comment (alisonjoseph) [inline services/web-app/pages/assets/designing/get-started/index.mdx:58]:
```suggestion
<MdxIcon name="sketch" />
```

File: services/web-app/pages/assets/designing/get-started/index.mdx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Get Axure"
+      aspectRatio="2:1"
+      actionIcon="launch"
+      href="https://gatsby.carbondesignsystem.com"
+      >
+
+![Sketch icon](/mdx/sketch-icon.png)
```
Comment (alisonjoseph) [inline services/web-app/pages/assets/designing/get-started/index.mdx:82]:
```suggestion
<MdxIcon name="sketch" />
```

**Review by alisonjoseph — APPROVED** [review]
LGTM

do we need these two files in the public directory?
sketch.svg
figma.svg

File: services/web-app/pages/designing/get-started.mdx [hunk 4]
```diff
@@ -1,6 +1,156 @@
...
+| [IBM Design Language](https://www.ibm.com/design/language/)                                                                 | IBM’s design philosophy and principles that govern and guide every experience designed by IBM.                                                                                |
+| [IBM Style](https://ibmdocs-test.mybluemix.net/docs/en/ibm-style?_ga=2.139068405.716538948.1649782752-687898146.1623775061) | A central reference that ensures all IBM content for external audiences is grammatically correct, clear, consistent, appropriate for global audiences, and easy to translate. |
+| [IBM Experience Standards](https://w3.ibm.com/design/experience-standards/)                                                 | Provides provides a shared definition of our standards at IBM, a way to measure excellence, and actionable ways to improve product or service experiences.                    |
```
Comment (francinelucca) [inline services/web-app/pages/designing/get-started.mdx:155]:
```suggestion
| [IBM Experience Standards](https://w3.ibm.com/design/experience-standards/)                                                 | Provides a shared definition of our standards at IBM, a way to measure excellence, and actionable ways to improve product or service experiences.                    |
```

**Review by francinelucca — DISMISSED** [review]
LGTM! minor typo CR, like Alison said, I think we can remove `services/web-app/public/designing/get-started/figma.svg` and `services/web-app/public/designing/get-started/sketch.svg`
is this ok to merge w.o. the DK table?

**Comment by andreancardona** [comment]
@francinelucca just made those changes. :)

**Comment by alisonjoseph** [comment]
> is this ok to merge w.o. the DK table?

I don't see why not

**Review by francinelucca — APPROVED** [review]
💃

**Pull request merged**